components/textit/actions/start-flow/start-flow.mjs

```diff
@@ -0,0 +1,50 @@
+import textit from "../../textit.app.mjs";
+
+export default {
+  name: "Start Flow",
+  version: "0.0.1",
+  key: "textit-start-flow",
+  description: "Starts a flow. [See docs here](https://textit.in/api/v2/flow_starts)",
+  type: "action",
+  props: {
+    textit,
+    flowUuid: {
+      propDefinition: [
+        textit,
+        "flowUuid",
+      ],
+    },
+    contactUuid: {
+      propDefinition: [
+        textit,
+        "contactUuid",
+      ],
+    },
+    groupUuid: {
+      propDefinition: [
+        textit,
+        "groupUuid",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const response = await this.textit.startFlow({
+      $,
+      data: {
+        flow: this.flowUuid,
+        contacts: [
```

Comment:
How about changing `contactUuid` type to `string[]` and using it as `contacts:this.contactUuid,`?